src/components/business-lookup/BusinessLookup.stories.ts

```diff
@@ -0,0 +1,38 @@
+import { BusinessLookup } from './index'
+
+export default {
+  title: 'component/BusinessLookup',
+  component: BusinessLookup,
+  argTypes: {
+  }
+}
+
+const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { BusinessLookup },
+  template: '<business-lookup v-bind="$props" />' // $props comes from args below
+})
+
+// sample Business Lookup result
+const result = {
+  identifier: 'BC1234567',
+  name: 'Test Business Name'
+}
+
+// mock Business Lookup Services class
+class BusinessLookupServices {
+  static async search (query: string): Promise<any[]> {
+    return Promise.resolve([result])
+  }
+}
+
+export const Default = Template.bind({})
+Default.args = {
+  showErrors: false,
+  businessLookup: {
+    identifier: result.identifier,
+    name: result.name
+  },
+  BusinessLookupServices,
+  hasBusinessLookupChanges: true
+}
```

Comment:
This component doesn't quite work correctly in Storybook because it relies on emitted events to update/reset the data.

Storybook doesn't handle this natively. I looked at `@storybook/addon-events` and `@storybook-auto-events` packages to include this but I didn't find any helpful examples, I did find some people reporting trouble using these, and after an hour I gave up. FWIW.